storey/drivers.py

```diff
@@ -158,7 +160,9 @@ async def _save_key(self, container, table_path, key, aggr_item, partitioned_by_
 
         if should_raise_error:
             raise V3ioError(
-                f'Failed to save aggregation for {table_path}/{key}. Response status code was {response.status_code}: {response.body}')
+                f'Failed to save aggregation for {table_path}/{key}. Response status code was {response.status_code}: {response.body}\n'
+                f'Update expression was: {update_expression}'
```

Comment:
why did you remove the update expr? it's easier to debug with it

Reply:
That's why I added it, not remove it 🤔

Reply:
lol :D